tests/dummy/app/pods/demo/controller.js

```diff
@@ -1,51 +1,29 @@
 import Ember from 'ember'
+import _ from 'lodash'
 
 export default Ember.Controller.extend({
-  data: [
-    {
-      'value': 'Clark Kent',
-      'label': 'Superman'
-    },
-    {
-      'value': 'Bruce Wayne',
-      'label': 'Batman'
-    },
-    {
-      'value': 'Tony Starks',
-      'label': 'Ironman'
-    },
-    {
-      'value': 'Princess Diana',
-      'label': 'Wonder Woman'
-    },
-    {
-      'value': 'Barry Allen',
-      'label': 'The Flash'
-    },
-    {
-      'value': 'Arthur Curry',
-      'label': 'Aquaman'
-    },
-    {
-      'value': 'Hal Jordan',
-      'label': 'Green Lantern'
-    },
-    {
-      'value': 'Adam Meadows',
-      'label': 'Atom'
-    },
-    {
-      'value': 'Jaime Reyes',
-      'label': 'Blue Beetle'
-    },
-    {
-      'value': 'Leonard Snart',
-      'label': 'Captain Cold'
+  data: Ember.computed('data', 'search', function () {
+    let result = this.model.map((record) => {
+      return {
+        label: record.get('label'),
+        value: record.id
+      }
+    })
+    if (this.get('search')) {
+      let filteredResult = _.filter(result, (item) => {
+        return item.label.toLowerCase().indexOf(this.get('search').toLowerCase()) !== -1
+      })
+      result = filteredResult
     }
-  ],
+    return result
+  }),
   actions: {
     onChange (values) {
-      console.log(values)
+      console.log('User selected: ' + values)
```

Comment:
Probably good to get in the practice of using `Ember.Logger.log()` instead of `console.log()`.